minio/api.py

```diff
@@ -139,7 +135,9 @@ class Minio(object):
     :return: :class:`Minio <Minio>` object
```

Comment:
Can we say about session_token param here?

Reply:
done